src/test/CommandSetTest.cpp

```diff
@@ -0,0 +1,86 @@
+#include "CommandSet.hpp"
```

Comment:
I am SO SO SO happy to see this! THANK YOU :)

Reply:
no trouble. I've not touched C++ properly for ages; it's great practice.